frontend/src/pages/auth/auth.tsx

```diff
@@ -27,6 +28,10 @@ const Auth: React.FC = () => {
     [dispatch],
   );
 
+  useEffect(() => {
+    void dispatch(authActions.getCurrentUser());
```

Comment:
Shouldn't we dispatch fetch current user only in single place?

Reply:
right, only in the app